doc/user-guide/Features-and-supported-standards.md

```diff
@@ -77,8 +77,8 @@
 |0352|[Client State Indication](http://www.xmpp.org/extensions/xep-0352.html)|`mod_csi`|
 |0357|[Push Notifications](http://www.xmpp.org/extensions/xep-0357.html)|`mod_event_pusher_push`|
 |0363|[HTTP File Upload](https://xmpp.org/extensions/xep-0363.html)|`mod_http_upload`|
-|0375|[XMPP Advanced Server 2017](https://xmpp.org/extensions/xep-0387.html) (without 0369: Mediated Information eXchange)|
 |0384|[OMEMO Encryption](https://xmpp.org/extensions/xep-0384.html) (MongooseIM supports PEP, which is required by this extension)||
+|0387|[XMPP Advanced Server 2018](https://xmpp.org/extensions/xep-0387.html)|
```

Comment:
But 0387 is "XMPP Compliance Suites 2018".

Can we name in "XMPP Compliance Suites: Advanced Server level" ?

Reply:
True. Perhaps the most correct statement would be: "XMPP Compliance Suite - all suites, Advanced Server level".